src/Middleware.cpp

```diff
@@ -1,5 +1,5 @@
-// OBECA - Open Broadcast Edge Cache Appliance
-// Gateway Process
+// 5G-MAG Reference Tools
+// MBMS Middleware Process
 //
 // Copyright (C) 2021 Klaus Kühnhammer (Österreichische Rundfunksender GmbH & Co KG)
```

Comment:
Not sure if this needs to be changed, maybe to `@author`

Reply:
AFAIK, the original copyright remains as it is.

Reply:
That aside, I'm not a fan of having dynamic auto-generated content like comitters, dates, etc. in the file comment headers, the diffs this creates make things harder to merge.